acceptance/analyzer_test.go

```diff
@@ -980,6 +1068,26 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 
 							assertLogsAndRestoresCacheMetadata(t, copyDir, output)
 						})
+
+						it("throws read/write error accessing cache image", func() {
+							h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7 does not validate cache flag")
+							cmd := exec.Command(
+								"docker", "run", "--rm",
+								"--network", registryNetwork,
+								"--env", "CNB_PLATFORM_API="+platformAPI,
+								"--name", containerName,
+								analyzeImage,
+								ctrPath(analyzerPath),
+								"-cache-image",
+								noAuthRegCacheImage,
+								noAuthRegistry.RepoName("some-image"),
```

Comment:
```suggestion
								readOnlyRegistry.RepoName("some-image"),
```

Reply:
We could also change the context ca. line 1030 to say `read only registry`...

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/c2e71adc20406f06b4f2d3d9b2626778faa93183)